Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2018"
 [dependencies]
 anyhow = "1"
 async-trait = "0.1"
+base64 = "0.12.3"
```

Comment:
I'd recommend not specifying the patch version if it's not needed as we are shipping a lib.